exercises/practice/list-ops/spec/list_ops_spec.cr

```diff
@@ -0,0 +1,119 @@
+require "spec"
+require "../src/*"
+
+describe "ListOps" do
+  describe "append entries to a list and return the new list" do
+    it "empty lists" do
+      list_ops = ListOps(Int32).new
+      list_ops.append([] of Int32, [] of Int32).should eq([] of Int32)
+    end
+    pending "list to empty list" do
+      list_ops = ListOps(Int32).new
+      list_ops.append([] of Int32, [1, 2, 3, 4]).should eq([1, 2, 3, 4])
+    end
+    pending "empty list to list" do
+      list_ops = ListOps(Int32).new
+      list_ops.append([1, 2, 3, 4], [] of Int32).should eq([1, 2, 3, 4])
+    end
+    pending "non-empty lists" do
+      list_ops = ListOps(Int32).new
+      list_ops.append([1, 2], [2, 3, 4, 5]).should eq([1, 2, 2, 3, 4, 5])
+    end
+  end
+  describe "concatenate a list of lists" do
```

Comment:
Can you update the template so that there are newlines between `describe`s?

```suggestion
  end

  describe "concatenate a list of lists" do
```